### PR TITLE
Add documentation for Tensorflow based Google bert xnli and bert base…

### PR DESCRIPTION
**Pull request by pallavigopal**
… Squad1.1 model fine tuning and IR generation

### CODE REVIEW

**Comment by pallavigopal** [comment]
@AlexanderDokuchaev Please review the PR

File: tensorflow_toolkit/nlp/bert/README.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+	$ cd tensorflow
+	$ python tensorflow/python/tools/freeze_graph.py \
+	--input_saved_model_dir=/tmp/xnli_output/export_model/<folder>/ \
+	--output_graph= ./bert_xnli_fp32_graph.pb \
+	--output_node_names=loss/LogSoftmax
+	```
+5. Run the OpenVino Model Optimizer to generate the IR
+
+	```bash
+	$ python $MO_DIR/mo.py --framework=tf \
```
Comment (AlexanderDokuchaev) [inline tensorflow_toolkit/nlp/bert/README.md:95]: Write as 
```
mo.py --framework=tf \
...
```
Directory with `mo.py` is added  to `PATH` by  `source /opt/intel/openvino/bin/setupvars.sh`.

File: tensorflow_toolkit/nlp/bert/README.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+	```bash
+	$ git clone https://github.com/google-research/bert
+	$ cd bert
+	```
+2. This folder contains a git patch to add support to export the saved model from the trained model for xnli and squad.
+   Apply the `0001-Add-support-to-export-the-saved-model-to-the-run_cla.patch` file to the google bert repository
+
+	```bash
+	$ git checkout eedf5716ce1268e56f0a50264a88cafad334ac61
+	$ git am 0001-Add-support-to-export-the-saved-model-to-the-run_cla.patch
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:22]: thanks for the review comments. will do

File: tensorflow_toolkit/nlp/bert/README.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+	```bash
+	https://github.com/google-research/bert/blob/master/multilingual.md
+	```
+	```bash
+	$ export BERT_BASE_DIR=/path/to/bert/chinese_L-12_H-768_A-12
+	$ export XNLI_DIR=/path/to/xnli
+	$ pip install tensorflow==1.14
+	```
+	```bash
+	$ python run_classifier.py \
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:43]: ok

File: tensorflow_toolkit/nlp/bert/README.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+	--train_batch_size=32 \
+	--learning_rate=5e-5 \
+	--num_train_epochs=2.0 \
+	--output_dir=/tmp/xnli_output \
+	--export_dir=/tmp/xnli_output/export_model
+	```
+4. Clone the tensorflow source and generate the frozen graph
+
+	```bash
+	$ git clone -b r1.14 --single-branch https://github.com/tensorflow/tensorflow.git
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:85]: ok i will check and update

File: tensorflow_toolkit/nlp/bert/README.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+	$ cd tensorflow
+	$ python tensorflow/python/tools/freeze_graph.py \
+	--input_saved_model_dir=/tmp/xnli_output/export_model/<folder>/ \
+	--output_graph= ./bert_xnli_fp32_graph.pb \
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:89]: ok

File: tensorflow_toolkit/nlp/bert/README.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
+# Fine tuning Bert base XNLI and Squad 1.1 from Google Bert
+
+## Prerequistes
+
+	*Ubuntu 18.02
+	*Tensorflow 1.1X
+	*OpenVINO 2020.1
+
+## TensorFlow BERT Installation
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:9]: ok

File: tensorflow_toolkit/nlp/bert/README.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+	--init_checkpoint=$BERT_BASE_DIR/model.ckpt-14599 \
+	--do_predict=True \
+	--predict_file=$SQUAD_DIR/dev-v1.1.json \
+	--train_batch_size=12 \
+	--learning_rate=3e-5 \
+	--num_train_epochs=2.0 \
+	--max_seq_length=384 \
+	--doc_stride=128 \
+	--output_dir=/tmp/squad_base/
+	--export_dir = /tmp/squad_base/export_model/
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:163]: ok

File: tensorflow_toolkit/nlp/bert/README.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+
+1. Download the pre-trained Bert-base chinese model
+
+	```bash
+	$ wget https://storage.googleapis.com/bert_models/2018_11_03/chinese_L-12_H-768_A-12.zip
+	```
+2. Refer the Fine-tuning Example section of the multilingual google wiki for the data set and finetuning
+
+	```bash
+	https://github.com/google-research/bert/blob/master/multilingual.md
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:35]: ok

File: tensorflow_toolkit/nlp/bert/README.md [hunk 9]
```diff
@@ -0,0 +1,206 @@
...
+5. Run the OpenVino Model Optimizer to generate the IR
+
+	```bash
+	$ python $MO_DIR/mo.py --framework=tf \
+	--input='input_ids_1,input_mask_1,segment_ids_1' \
+	--output='loss/LogSoftmax' \
+	--input_model=fp32_model.pb \
+	--output_dir=fp32/ \
+	--input_shape=[1,128],[1,128],[1,128] \
+	--log_level=DEBUG --disable_nhwc_to_nch
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:101]: ok

File: tensorflow_toolkit/nlp/bert/README.md [hunk 10]
```diff
@@ -0,0 +1,206 @@
...
+
+1. Download the Pre-trained Bert base uncased model
+	```bash
+	$ wget https://storage.googleapis.com/bert_models/2018_10_18/uncased_L-12_H-768_A-12.zip
+	```
+
+2.  Refer the SQuad1.1 section of the google wiki for the data set and finetuning
+
+	```bash
+	Download the below data to some directory $SQUAD_DIR
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:114]: ok

File: tensorflow_toolkit/nlp/bert/README.md [hunk 11]
```diff
@@ -0,0 +1,206 @@
+# Fine tuning Bert base XNLI and Squad 1.1 from Google Bert
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:1]: ok sure

File: tensorflow_toolkit/nlp/bert/README.md [hunk 12]
```diff
@@ -0,0 +1,206 @@
...
+	--max_seq_length=384 \
+	--doc_stride=128 \
+	--output_dir=/tmp/squad_base/
+	```
+	The dev set predictions will be saved into a file called predictions.json in the output_dir:
+	```bash
+	$ python $SQUAD_DIR/evaluate-v1.1.py $SQUAD_DIR/dev-v1.1.json /tmp/squad_base/predictions.json
+	```
+	Results:
+		{"exact_match": 81.17313150425733, "f1": 88.49906696207893}
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:146]: ok

File: tensorflow_toolkit/nlp/bert/README.md [hunk 13]
```diff
@@ -0,0 +1,206 @@
...
+	--export_dir = /tmp/squad_base/export_model/
+	```
+
+4. Clone the tensorflow source and generate the frozen graph
+
+	```bash
+	$ git clone -b r1.14 --single-branch https://github.com/tensorflow/tensorflow.git
+	$ cd tensorflow
+	$ python tensorflow/python/tools/freeze_graph.py \
+	--input_saved_model_dir=/tmp/squad_base/exported_model/<>/ \
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:172]: use the director created during training

File: tensorflow_toolkit/nlp/bert/README.md (hunk 1 above)
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:95]: ok

**Comment by pallavigopal** [comment]
@AlexanderDokuchaev  Thanks for your review comments. I will address the comments ASAP

File: tensorflow_toolkit/nlp/bert/README.md [hunk 14]
```diff
@@ -0,0 +1,206 @@
...
+	*Ubuntu 18.02
+	*Tensorflow 1.1X
+	*OpenVINO 2020.1
+
+## TensorFlow BERT Installation
+
+1. Clone the google bert
+
+	```bash
+	$ git clone https://github.com/google-research/bert
```
[the remark this answers is not in the document]
Comment (pallavigopal) [reply tensorflow_toolkit/nlp/bert/README.md:14]: ok

**Comment by pallavigopal** [comment]
@AlexanderDokuchaev  Address the review comments. Please review the changes

**Pull request merged**